bot/exts/evergreen/githubinfo.py

```diff
@@ -0,0 +1,88 @@
+import logging
+from datetime import datetime
+
+import aiohttp
+import discord
+from discord.ext import commands
+from discord.ext.commands.cooldowns import BucketType
+
+log = logging.getLogger(__name__)
+
+
+class GithubInfo(commands.Cog):
+    """Fetches GitHub."""
```

Comment:
```suggestion
    """Fetches info from GitHub."""
```
"Fetches GitHub" seems a bit off to me.